pkgs/development/libraries/ogre/default.nix

```diff
@@ -34,12 +54,13 @@ stdenv.mkDerivation rec {
      libXxf86vm libICE
      libXrender
      SDL2
+     pugixml
    ] ++ lib.optional withNvidiaCg nvidia_cg_toolkit;
 
   meta = {
```

Comment:
```suggestion
  meta = with lib; {
```

Reply:
If we are redoing large parts of the package anyway we can also clean this up.

Reply:
@luc65r: Sounds like you might have some bigger rework plans, but I put some "fixup" patches based on @SuperSandro2000's suggestions into this branch. I also had to bump the mysocketw version to 3.10.27 to get it to build with openssl.

https://github.com/jljusten/nixpkgs/commits/rigsofrods-2021.04